CMakeLists.txt

```diff
@@ -33,11 +35,16 @@ if(WIN32 AND BUILD_SHARED_LIBS)
   set(BUILD_SHARED_LIBS OFF CACHE BOOL "" FORCE)
 endif()
 
+# build options
+option(K2_USE_GLOG "Flag that chooses logger from [GLOG, NON-GLOG (loguru)]" OFF)
```

Comment:
Why is this off by default?  Since we install gtest by default, doesn't that already require glog?

Reply:
@songmeixu 
when to enable this option?

Reply:
OFF is set for test, and help figure out what we need for logging that is outside a minimal one. 
The gtest doesn't depend on glog, the former is for unit test, while the latter is for logging. Though the implementations are similar and very simple for some cases (*_EQ), I think there is a design principle to isolate these two.